stablehlo/integrations/python/tests/stablehlo_to_tf_saved_model_test.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 The StableHLO Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mlir.dialects.stablehlo as stablehlo
+import mlir.ir as ir
+from mlir.stablehlo_to_tf_saved_model import InputLocation, save_stablehlo_as_tf_saved_model
```

Comment:
This doesn't make sense to me. If possible, I think we want it to be:

```
from stablehlo.savedmodel import stablehlo_to_tf_saved_model
```

But lets get it working first.

Reply:
I am not sure if we should put the `savedmodel` under `mlir.dialects`.
Currently the package is hoisted at `build/python_packages/stablehlo/savedmodel` which allows me to import it as:
 
```
from savedmodel.stablehlo_to_tf_saved_model import InputLocation, stablehlo_to_tf_saved_model
```

Please let me know what do you think?

Reply:
With the recent changes now the import looks like
```
from mlir.stablehlo.savedmodel.stablehlo_to_tf_saved_model import InputLocation, stablehlo_to_tf_saved_model
```
Please let me know your opinion.

Just to document:
Adding the nesting `stablehlo/savedmodel` parallel to mlir gives error like
```
ModuleNotFoundError: No module named 'stablehlo.savedmodel'; 'stablehlo' is not a package
```

Reply:
```
from mlir.stablehlo.savedmodel.stablehlo_to_tf_saved_model import InputLocation, stablehlo_to_tf_saved_model
```

LGTM!

Reply:
Yup this is what we have added now in the test file.